### PR TITLE
fix and enable ImportFlowForm component test

### PR DESCRIPTION
**Pull request by vikram-raj**
Enable and if ImportSlowForm component unit test.

### CODE REVIEW

File: frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx [hunk 1]
```diff
@@ -84,80 +78,98 @@ describe('ImportFlowForm: ', () => {
...
-    importFlowForm.find('input[data-test-id="import-git-repo-url"]').simulate('change', {
+  it('detect git type', () => {
+    const importFlowForm = shallow(<ImportFlowForm {...props} />);
+    importFlowForm.find('[data-test-id="import-git-repo-url"]')
+      .shallow().find('[data-test-id="import-git-repo-url"]').simulate('change', {
       target: { value: 'https://github.com/vikram-raj/console/tree/import-flow' },
     });
-    importFlowForm.find('input[data-test-id="import-git-repo-url"]').simulate('blur');
+    importFlowForm.find('[data-test-id="import-git-repo-url"]')
+    .shallow().find('[data-test-id="import-git-repo-url"]').simulate('blur');
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx]: So much duplication of `find('[data-test-id="import-git-repo-url"]')`
Why are you calling `.shallow()` again?

File: frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx [hunk 2]
```diff
@@ -84,80 +78,90 @@ describe('ImportFlowForm: ', () => {
...
-    importFlowForm.find('input[data-test-id="import-git-repo-url"]').simulate('change', {
+    urlInput.simulate('change', {
       target: { value: 'http://localhost:9000/dev/add' },
     });
-    importFlowForm.find('input[data-test-id="import-git-repo-url"]').simulate('blur');
-    expect(importFlowForm.state().gitRepoUrlError).toBe('Please enter the valid git URL');
-    expect(importFlowForm.state().gitTypeDetected).toBe(false);
+    urlInput.simulate('blur');
+    expect(importFlowForm.state().gitRepoUrlError).toBe('Please enter a valid git URL.');
+    expect(importFlowForm.state().gitTypeDetected).toBe(undefined);
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx]:
```suggestion
    expect(importFlowForm.state().gitTypeDetected).toBeUndefined();
```

File: frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx [hunk 3]
```diff
@@ -84,80 +78,90 @@ describe('ImportFlowForm: ', () => {
...
   });
 
-  xit ('form submission', () => {
-    const importFlowForm = mount(<ImportFlowForm {...props} />);
-    importFlowForm.find('input[data-test-id="import-git-repo-url"]').simulate('change', {
+  it ('form submission', () => {
+    const preventDefault = jest.fn();
+    const importFlowForm = shallow(<ImportFlowForm {...props} />);
+    importFlowForm.find('[data-test-id="import-git-repo-url"]')
+      .shallow().find('[data-test-id="import-git-repo-url"]').simulate('change', {
```
Comment (christianvogt) [inline frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx]: Why are you calling `.shallow()` ?

Comment (vikram-raj) [reply frontend/public/extend/devconsole/components/ImportFlowForm/__tests__/ImportFlowForm.spec.tsx]: fixed it.```.shallow()``` is not required here.

**Review by christianvogt — APPROVED** [review]

**Pull request merged**